lib/cli/command.ts

```diff
@@ -22,6 +22,11 @@ export const callback = (error: Error | null, response: grpcResponse) => {
   if (error) {
     console.error(`${error.name}: ${error.message}`);
   } else {
-    console.log(JSON.stringify(response.toObject(), undefined, 2));
+    const responseObj = response.toObject();
+    if (Object.keys(responseObj).length === 0) {
+      console.log('success');
```

Comment:
Just printing `success` is probably not the the most beautiful thing to do. It is fine for now but we will have to make the cli more beautiful and appealing eventually.

Reply:
Yup, but I figure that can be handled along with other cli user friendliness/aesthetics improvements.